i18n.module

```diff
@@ -422,15 +422,15 @@ function i18n_object($type, $object) {
 /**
  * Get object wrapper by object key.
  *
- * @param $type
+ * @param string $type
  *   The object type to load e.g. node_type, menu, taxonomy_term.
- * @param $key
+ * @param string $key
  *   The object key, can be an scalar or an array.
- * @param $object
+ * @param object $object
```

Comment:
If beliefing the comment line below, this can be object or array. Needs evaluation.